src/screens/Profile/TravelToken/use-token-toggle-details.ts

```diff
@@ -13,8 +13,13 @@ const useTokenToggleDetails = (shouldFetchTokenDetails: boolean) => {
     const toggleToggleDetails = await getTokenToggleDetails();
     if (toggleToggleDetails) {
       const {toggleMaxLimit, toggledCount} = toggleToggleDetails;
-      if (toggleMaxLimit && toggleMaxLimit >= toggledCount) {
-        setToggleLimit(toggleMaxLimit - toggledCount);
+      if (toggleMaxLimit) {
+        if (toggleMaxLimit >= toggledCount) {
+          setToggleLimit(toggleMaxLimit - toggledCount);
+        } else {
+          //We can end up here if we decide to reduce max limit value in firestore
+          setToggleLimit(0);
+        }
```

Comment:
Don't have to create a PR for changing it, but it might be a little clearer something like `Math.max(toggleMaxLimit - toggledCount, 0)`